test/integration/fixtures/parallel/test3.mjs

```diff
@@ -0,0 +1,7 @@
+import {describe,it} from "../../../../index.js";
+
+describe('test3', () => {
+  it('should fail', () => {
+    throw new Error('expecting this error to fail');
```

Comment:
@nicojs  hello! I was reviewing your PR, and have a question for this change.
You made three differenct tests, test1, test2, test3. Can you explain about this tests?
Why test1 and test2 pass whereas test3 fail? Also Why do we need test1 and test2 each, when they have the same dir path and they both pass. Thank you 👩